code-challenges.test.js

```diff
@@ -65,5 +175,42 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
-
-// b) Create the function that makes the test pass.
+describe ("firstVowel", () => {
+    it("takes in a string and logs the index of the first vowel", 
+    () => {
+      expect(firstVowel(vowelTester1)).toEqual(1)
+      expect(firstVowel(vowelTester2)).toEqual(0)
+    })
+  })
+
+//   ReferenceError: firstVowel is not defined
+
+
+// // b) Create the function that makes the test pass.
+
+
+// Pseudo Coding
+// first I am going to create a function called firtVowel
+// Then I am going to have it take in a string loop through it so it can idenitfy the first vowel of the value
+// I will have it return a new string with new values
+// I recived help with this question
```

Comment:
Awesome!  Reaching out for help is always encouraged as long as you are able to understand the code that is turned in.